app/controllers/v1/recipes_controller.rb

```diff
@@ -26,9 +28,9 @@ def show
   def update
     recipe = Recipe.find(params[:id])
     if recipe.update(recipe_params)
-      render json: { message: 'Your recipe has been updated.' }, status: 200
+      render json: { message: 'Your recipe has been updated.' }, status: 201
```

Comment:
please change this to a 200 response

Reply:
Yeah, I agree. on second thought, it should be 200